modular_citadel/code/modules/mob/cit_emotes.dm

```diff
@@ -194,4 +194,19 @@
 			user.apply_damage(20, BRUTE, def_zone)
 		if(luck >= 95)
 			user.adjustBrainLoss(100)
-	. = ..()
\ No newline at end of file
+	. = ..()
+
+
+/datum/emote/living/mothsqueak
+	key = "msqueak"
+	key_third_person = "lets out a tiny squeak"
+	message = "lets out a tiny squeak!"
+	emote_type = EMOTE_AUDIBLE
+	mob_type_allowed_typecache = list(/mob/living/carbon)
+
+/datum/emote/living/mothsqueak/run_emote(mob/living/user, params)
+		if(user.nextsoundemote >= world.time)
```

Comment:
this entire block needs to be shifted down in terms of indentation so this compiles